apps/core/priv/scaffolds/terraform/providers/kind.eex

```diff
@@ -0,0 +1,25 @@
+terraform {
+  backend "local" {
+    path = "../../{{ .Values.Bucket }}/{{ .Values.__CLUSTER__ }}/{{ .Values.Prefix }}/terraform.tfstate"
+  }
+
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "~> 2.8.0"
+    }
+  }
+}
+
+{{- if .Values.ClusterCreated }}
```

Comment:
shouldn't this be the inverse?

Reply:
Copied from the Equinix setup. `ClusterCreated` seems to be if the cluster is being deployed during that step. The wording is a bit confusing though, I remembered having it reversed at first with Equinix.